portfolio/src/main/webapp/js/script.js

```diff
@@ -31,12 +31,17 @@ function addRandomFact(id) {
   factContainer.innerHTML = fact;
 }
 
-
-function validate() {
-    var phone = $("#telephone").val();
-    if (phone.length != 10 || isNaN(phone)) {
-        alert("Phone  number is not valid or not valid format.");
-        return false;
-    }
+function validatePhoneNumber() {
+  var phone = $("#telephone").val();
+  var phoneNum = /^[+]?(1\-|1\s|1|\d{3}\-|\d{3}\s|)?((\(\d{3}\))|\d{3})(\-|\s)?(\d{3})(\-|\s)?(\d{4})$/g;
+  if(phone.match(phoneNum))
+  {
+    document.getElementById('invalid').innerHTML = "<p style='color:gray'> Phone Number</p>";
+	return true;
+  }
+  else
+  {	   
+    document.getElementById('invalid').innerHTML = "<p style='color:red'> Phone Number</p>";
+	return false;
```

Comment:
Move Line 45 to left by two spaces.

Reply:
I don't know why it is appearing like that on github when locally they are the same, but I deleted it and redid the indentation locally. Hopefully this fixes the issue.